### ISSUE TITLE
[ImageClassifier] Add option to calculate softmax from the output of the model

### ISSUE THREAD

**Issue by jfix71** [pytorch/glow#1947]
Inference models may skip the final softmax op of the network. This means the probabilities printed by the ImageClassifier are incorrect, as it expects softmax to be the final op. We could calculate the softmax in the ImageClassifier itself. This would allow the user to see the probabilities even if the network is missing it. This would likely make sense as a command line option.

**Comment by rdzhabarov** [comment]
For testing purposes, you could use an existing model from Glow model zoo: densenet121
